scripts/generate.sh

```diff
@@ -0,0 +1,14 @@
+echo "Regenerating account client"
+CLIENT_DIRECTORY="../src/generated-rest"
+
+echo "Deleting contents of $CLIENT_DIRECTORY"
+
+rm -rf $CLIENT_DIRECTORY
+
+echo "Generating account client..."
+openapi-generator-cli generate -g typescript-axios -o $CLIENT_DIRECTORY -i ./src/openapi.json --additional-properties=supportsES6=true,stringEnums=true
+
+echo "Remove unnecessary generator file"
+rm openapitools.json
+
+echo "Done!"
```

Comment:
```suggestion
echo "Done!"

```

VSCode has a setting to automatically add newlines to saved files.

```
files.insertFinalNewline
```

A lot of files in `ui-shared` already have a final newline; I think we should do it for all new files, for consistency.